test/test_bullet_world.py

```diff
@@ -169,6 +169,71 @@ def test_no_object_found_for_given_prospection_object(self):
             self.assertTrue(True)
         time.sleep(0.05)
 
+    def test_real_object_position_does_not_change_with_prospection_object(self):
+        milk_2_pos = [1.3, 1, 0.9]
+        milk_2 = Object("milk", ObjectType.MILK, "milk.stl", pose=Pose(milk_2_pos))
```

Comment:
Can you change the name of the Object to "milk2". In PR #155 I enforce Unique names, therefore this test would fail.

Reply:
sure